frontend/ui/measuretable.jsx

```diff
@@ -25,6 +25,10 @@ const getReleaseValue = (release, countType, timeWindow) => {
 
 class MeasureTable extends React.Component {
   render() {
+    if (this.props.measures.length === 0) {
```

Comment:
Returning `null` to not display something is kind of counterintuitive. I actually think the conditional here should be on what's rendering `MeasureTable` (subview in this case), probably around here: https://github.com/mozilla/missioncontrol/blob/5daea6f89ed5b780aa553b0cffdae06857a9341f/frontend/ui/subview.jsx#L166

Could you back this out (along with the test)? Perhaps we can write some tests for subview when modifying that...

Reply:
I have added the check in `subview.jsx`. I'll add the test for this along with other tests for subview in a later PR!

Reply:
@wlach fixed. I'll open another PR for handling #335. Please review.